pythonforandroid/build.py

```diff
@@ -374,7 +365,6 @@ def prepare_build_environment(self,
         if not self.ccache:
             info('ccache is missing, the build will not be optimized in the '
                  'future.')
-        self.cython = get_cython_path()
```

Comment:
Could we still add a check here for Cython being available, or a BuildInterruptingException if not?

Reply:
Oh, good point! Sure, that would be possible, let me have a look at it